src/dex/fluid-dex/fluid-dex.ts

```diff
@@ -154,10 +154,10 @@ export class FluidDex extends SimpleExchange implements IDex<FluidDexData> {
     return pool ? [pool.id] : [];
```

Comment:
in `getPoolIdentifiers` we can get rid of `await` before `getPoolByTokenPair` call as well

Reply:
@KanievskyiDanylo fixed